src/protocol-parser/protocol-parser.ts

```diff
@@ -20,8 +20,6 @@
  * Parser types should match the `../protocol` types.
  */
 
-const MAX_INT = 9007199254740991 as const;
```

Comment:
NIT: Strictly speaking we cannot rely on `Number.MAX_SAFE_INTEGER` to always be equal to the const [defined in BiDi spec](https://w3c.github.io/webdriver-bidi/#protocol-definition). But I cannot see any reasonable-enough reason why we want to change it in the protocol.

Reply:
Oh, that's surprising, but makes sense.

> But I cannot see any reasonable-enough reason why we want to change it in the protocol.

Why?